disks-and-volumes.md

```diff
@@ -21,6 +21,10 @@ A disk can be made accessible via four different types:
 All possible configuration options are available in the
 [Disk API Reference](https://kubevirt.github.io/api-reference/master/definitions.html#_v1_disk).
 
+All types but **floppy** allow you to specify the `Bus` attribute. The `Bus` attribute determine
+how the disk will be presented to the Guest Operating System. **floppy** disks don't support
+the `Bus` attribute: they are always attached to the `fdc` bus.
```

Comment:
Is fdc a bus or a device name?

I suppose that bus would be IDE, not?

Reply:
fdc is a bus name (related doc: https://libvirt.org/formatdomain.html#elementsControllers)

floppies have their own bus and controller, AFAIK different from IDE. Se also https://en.wikipedia.org/wiki/Floppy-disk_controller and c

Reply:
Wow - Thank you, I was never aware of this!